tests/dashboard/plot_orchestrator_test.py

```diff
@@ -1067,13 +1143,8 @@ def test_update_config_to_different_workflow_id_unsubscribe_old_subscribe_new(
         )
         plot_orchestrator.update_plot_config(cell_id, new_config)
 
-        # Should no longer be subscribed to old workflow
-        assert len(fake_job_orchestrator._workflow_subscriptions[workflow_id]) == 0
```

Comment:
Many asserts have been removed. In some tests there are none left.
Are we now just checking that nothing raises? Is it enough?